Modules/Numerics/Statistics/test/itkHistogramTest.cxx

```diff
@@ -844,12 +710,6 @@ itkHistogramTest(int, char *[])
     std::cout << "STREAMED ENUM VALUE HistogramToRunLengthFeaturesFilterEnums::RunLengthFeature: " << ee << std::endl;
   }
 
-  if (!pass)
-  {
-    std::cout << "Test failed in " << whereFail << '.' << std::endl;
-    return EXIT_FAILURE;
-  }
-
   std::cout << "Test passed." << std::endl;
```

Comment:
`pass` can have the value 'EXIT_FAILED' and the message "Test passed." may be misleading.

Reply:
This is the reason why I have been changing such messages to "Test finished." as time has permitted: https://github.com/InsightSoftwareConsortium/ITK/pull/4203/commits/dfc9fcd32fc6a18e6b333159b6ae3c3e93684a64#diff-7d75c318ecedf7ef490900a9487a737074445a8264da10f6f0f3ed459944cf21R853 One of the reasons of why the file has conflicts now: https://github.com/InsightSoftwareConsortium/ITK/pull/4205/conflicts. I will fix them later tonight.